cli/asconfig.js

```diff
@@ -68,7 +66,7 @@ function getASConfig(asconfigPath, targetName, readFile, cliOptions, required) {
     getASConfig(
       path.isAbsolute(config.extends)
         ? config.extends
-        : path.join(baseDir, config.extends),
+        : path.join(path.dirname(asconfigPath), config.extends),
```

Comment:
Wondering if doing a `resolve` instead of a `join` here would give us support for absolute extends path?

Reply:
I am not very good at working with node paths. Do you think that `path.resolve(dirname, config)` would be better?